src/pages/derivez/_get-started-steps-section.tsx

```diff
@@ -89,22 +89,22 @@ const GetStartedStepsSection: React.FC<TProps> = ({ is_demo = false, chosen_tab
                     />
                 ),
                 image: data['demo_step1'],
-                alt: <Localize translate_text={demo_step1_alt} />,
+                alt: demo_step1_alt,
```

Comment:
Are we sure these does not need to be localize?

Reply:
@yashim-deriv These are image alt message, AFAIK we don't need to translate them, as I checked in the code they didn't translate it in most of the places

Reply:
I asked SEO team about it, we need translations for our alt messages, will update my PR :pray: